content/pages/en/meetups.rst

```diff
@@ -17,16 +17,27 @@ Meetups
 Meetups
 ~~~~~~~
 
-We have monthly meetups with talks and workshop on various python related and development in general and data topics.
+In our monthly Meetup you can look forward to workshops and talks on various
+Python topics, e.g. from the field of software development or data science.
 
-You can find and join upcoming events on our `Meetup group <https://www.meetup.com/PyLadies-Berlin/>`__.
+In addition, we start our setup events with simple topics that will make your
```

Comment:
I would say, "In addition, we cover basic topics in our setup events that will ..."